File: SimpleCalculator/app/src/main/java/com/mkaraca/simplecalculator/MainActivity.java
package com.mkaraca.simplecalculator;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    EditText numberText1;
    EditText numberText2;
    TextView resultText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
         numberText1=findViewById(R.id.numberText1);
         numberText2=findViewById(R.id.numberText2);
         resultText=findViewById(R.id.resultText);
    }
    public void sum(View view){
        if(numberText1.getText().toString().matches("")||numberText2.getText().toString().matches("")){
            resultText.setText("Enter number!");
        }
        else {
            int number1 = Integer.parseInt(numberText1.getText().toString());
            int number2 = Integer.parseInt(numberText2.getText().toString());
            int result = number1 + number2;
            resultText.setText("Result: " + result);
        }
    }
    public void deduct(View view){
        if(numberText1.getText().toString().matches("")||numberText2.getText().toString().matches("")){
            resultText.setText("Enter number!");
        }
        else {
            int number1 = Integer.parseInt(numberText1.getText().toString());
            int number2 = Integer.parseInt(numberText2.getText().toString());
            int result = number1 - number2;
            resultText.setText("Result: " + result);
        }

    }
    public void multiply(View view){
        if(numberText1.getText().toString().matches("")||numberText2.getText().toString().matches("")){
            resultText.setText("Enter number!");
        }
        else {
            int number1 = Integer.parseInt(numberText1.getText().toString());
            int number2 = Integer.parseInt(numberText2.getText().toString());
            int result = number1 * number2;
            resultText.setText("Result: " + result);
        }

    }
    public void divide(View view){
        if(numberText1.getText().toString().matches("")||numberText2.getText().toString().matches("")){
            resultText.setText("Enter number!");
        }
        else {
            int number1 = Integer.parseInt(numberText1.getText().toString());
            int number2 = Integer.parseInt(numberText2.getText().toString());
            int result = number1 / number2;
            resultText.setText("Result: " + result);
        }
    }
}

File: tabyalar/app/src/main/java/com/mkaraca/tabyalar/MainActivity.kt
package com.mkaraca.tabyalar

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.FrameLayout

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<FrameLayout>(R.id.frame00).setOnClickListener{
            findViewById<FrameLayout>(R.id.frame00).visibility=View.INVISIBLE
            findViewById<FrameLayout>(R.id.frame11).visibility=View.VISIBLE
            findViewById<FrameLayout>(R.id.frame12).visibility=View.VISIBLE
        }
        findViewById<FrameLayout>(R.id.frame11).setOnClickListener {
            findViewById<FrameLayout>(R.id.frame11).visibility=View.INVISIBLE
            findViewById<FrameLayout>(R.id.frame12).visibility=View.INVISIBLE

        }
    }
}